content/docs/source-git/work-with-source-git/pull-upstream-fixes.md

```diff
@@ -29,12 +29,11 @@ configure the whole process.
 As an example, let's try to do this with systemd, in the example below we'd be
 using https://gitlab.com/packit-service/src/systemd. The remote mimics how
 `fedpkg fork` works.
-```
-$ git remote -v
-origin      git@gitlab.com:packit-service/src/systemd.git (push)
-ttomecek    git@gitlab.com:TomasTomecek/systemd-stable.git (push)
-upstream    git@github.com:systemd/systemd-stable.git (push)
-```
+
+    $ git remote -v
+    origin      git@gitlab.com:packit-service/src/systemd.git (push)
+    ttomecek    git@gitlab.com:TomasTomecek/systemd-stable.git (push)
+    upstream    git@github.com:systemd/systemd-stable.git (push)
```

Comment:
I find it funny how you hate triple-ticks :D

Reply:
I don't hate them, they're just redundant and in my opinion, this looks better even in markdown (it stands out when I skim through the text)